exotations/task_maps/task_map/include/task_map/LookAt.h

```diff
@@ -0,0 +1,60 @@
+/*
+ *      Author: Christopher E. Mower
+ *
+ * Copyright (c) 2018, University of Edinburgh
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ *  * Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *  * Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ *  * Neither the name of  nor the names of its contributors may be used to
+ *    endorse or promote products derived from this software without specific
+ *    prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+#ifndef LOOKAT_H_
+#define LOOKAT_H_
+
+#include <exotica/TaskMap.h>
+#include <task_map/LookAtInitializer.h>
+
+namespace exotica
```

Comment:
Doxygen comments explaining how the taskmap works with reference to pictures are missing (such that it shows nicely in the to-be-generated online documentation).

Reply:
How to go about adding images/math with doxygen in .cpp/.h source?

Reply:
Ok thanks, do we place images in a specific location in `exotica/doc` that the `.cpp`/`.h` doxygen  will find? I.e., the answer [here](https://stackoverflow.com/questions/30863422/doxygen-how-to-embed-images-on-main-page-for-all-output-formats) says:

> You can use shorter paths if you put your images in an image folder and specify it's location in your configuration: 
> ```IMAGE_PATH     = my_image_dir```

Do we have a pre-specified  ```IMAGE_PATH```?

Reply:
We have not yet defined an `IMAGE_PATH`. I am happy for you to add a `doc/image` folder to task_map or to have the image in the header or source directory in task_map as well - or in the exotica/doc.